File: azure/transporterSolution/transporterService/DataObjects/UserItem.cs
using Microsoft.WindowsAzure.Mobile.Service;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using transporterService.DataTranspObjects;

namespace transporterService.DataObjects
{
    public partial class UserItem : BaseItem
    {
        public string Name { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }

        //public string UserFacebookItemID { get; set; }
        //[ForeignKey("UserFacebookItemID")]
        public virtual UserFacebookItem UserFacebookItem {get;set;}
    }
}

File: android/app/src/main/java/com/example/fugliara/ccs002/server/dataobjects/UserItem.java
package com.example.fugliara.ccs002.server.dataobjects;

public class UserItem {

    @com.google.gson.annotations.SerializedName("id")
    public String id;
    @com.google.gson.annotations.SerializedName("name")
    public String name;
    @com.google.gson.annotations.SerializedName("email")
    public String email;
    @com.google.gson.annotations.SerializedName("password")
    public String password;
    @com.google.gson.annotations.SerializedName("userFacebookItem")
    public UserFacebookItem facebookItem;

    public UserItem() {
        id = "";
        name = "";
        email = "";
        password = "";
        facebookItem = null;
    }
}
